sdk/trace/span_test.go

```diff
@@ -243,3 +244,20 @@ func TestTruncateAttr(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkRecordingSpanSetAttributes(b *testing.B) {
```

Comment:
Can you add a call to https://pkg.go.dev/testing#B.ReportAllocs?

Reply:
Will do.